src/main/java/computer_science_basics/homework_2/Heghine_Khachatryan/Task3.java

```diff
@@ -258,15 +368,20 @@ public static void fibonacciSeries() {
 
         System.out.print(firstTerm + " " + secondTerm + " ");
 
-        // TODO: Your code here
+        for (int i = 0; i <= number; i++) {
+            thirdTerm = firstTerm + secondTerm;
+            System.out.println(thirdTerm);
+            firstTerm = thirdTerm;
+            secondTerm = console.nextInt();
```

Comment:
are you sure about line 375? 

may be you mean  

System.out.print( " " +  thirdTerm);
firstTerm = secondTerm;
secondTerm = thirdTerm;